views/parts/events/row.php

```diff
@@ -14,6 +14,31 @@
 $date_format = get_option( 'date_format' );
 $time_format = get_option( 'time_format' );
 
+$args_length       = 0;
+$parsed_args       = array();
+$show_args_preview = false;
+
+foreach ( $event->args as $arg ) {
+	if ( is_array( $arg ) || is_bool( $arg ) || is_object( $arg ) ) {
+		$parsed_args[] = array(
+			'type' => gettype( $arg ),
+			'msg'  => wp_json_encode( $arg ),
+		);
+	} else {
+		$parsed_args[] = array(
+			'type' => gettype( $arg ),
+			'msg'  => $arg,
+		);
+	}
+
+
+	$show_args_preview = is_array( $arg ) || is_object( $arg );
+}
+
+$args_length = array_sum( array_map( function( $ar ) {
+	return strlen( $ar['msg'] );
+}, $parsed_args ) );
```

Comment:
Please move this logic to the method that requires this view, pass the output as a view argument